analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/SETestContext.cs

```diff
@@ -83,16 +82,12 @@ public static SETestContext CreateVB(string methodBody, string additionalParamet
         {
             var code = $@"
 Public Class Sample
-
```

Comment:
These empty lines should not be removed from the VB sample.

After `Class` and before `End Class`:
https://github.com/SonarSource/sonar-dotnet/blob/master/docs/coding-style.md#vbnet-specifics

There should be empty line between methods.